vitessce/repr.py

```diff
@@ -32,11 +32,21 @@ def make_repr(init_locals, class_def=None):
 
     # Remove redundant constructor parameters (when the value equals the default value).
     for k, v in inspect.signature(clazz).parameters.items():
+        if k not in init_locals:
+            continue
+
         try:
-            if k in init_locals and init_locals[k] == v.default:
-                del init_locals[k]
-        except:
+            is_default = init_locals[k] == v.default
+        except NotImplementedError:
             # Equality comparison may not be implemented for the value object.
+            is_default = False
+
+        try:
+            if is_default:
+                del init_locals[k]
+        except ValueError:
+            # TODO: Is this also expected?
+            # ValueError('The truth value of a DataFrame is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().')
```

Comment:
Thanks, @keller-mark. I don't really grasp the connection between your comment and this code, if this is the part you're talking about: 
> it does not make sense to put a whole data frame in a repr - SnapWrapper depends on those dataframes passed in its constructor, rather than more simple values

I'll go ahead and merge this, but file a new issue to clean up the TODO.

Reply:
[Here](https://github.com/vitessce/vitessce-python/blob/1e100e4f3f6b2389a899552dffe90716ffafc6d5/vitessce/wrappers.py#L590) (within line 592, in the call to `make_repr`), `locals()` includes variables whose values are Pandas dataframes. In the `if` statement that you added in this PR ([`if is_default`](https://github.com/vitessce/vitessce-python/blob/1e100e4/vitessce/repr.py#L45)), `is_default` is one of those local variables, and therefore Python tries to use a dataframe as a truthy value in this case when the value is a dataframe.